### PR TITLE
feat(temporary-voter-registry-api): First version

### PR DESCRIPTION
**Pull request by stefanhar**
# \<Description\>

The temporary voter registry is provided to us as a xlsx file form the national registry.
We want to be able to query the temporary voter registry when creating new endorsements and when assigning voting region in the application system.

## What

- Added database config for temporary voter registry
- Added test
- Added api

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/temporary-voter-registry-api/src/app/modules/voterRegistry/voterRegistry.controller.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Controller, Get, NotFoundException, Query } from '@nestjs/common'
+import { FindOneDto } from './dto/findOne.dto'
+import { VoterRegistry } from './voterRegistry.model'
+import { VoterRegistryService } from './voterRegistry.service'
+
+@Controller('voter-registry')
+export class VoterRegistryController {
+  constructor(private readonly voterRegistryService: VoterRegistryService) {}
+
+  @Get()
```
Comment (eirikurn) [inline apps/services/temporary-voter-registry-api/src/app/modules/voterRegistry/voterRegistry.controller.ts:10]: Authorization?

**Review by eirikurn — COMMENTED** [review]
Similar review as #3872

File: apps/services/temporary-voter-registry-api/src/app/modules/voterRegistry/voterRegistry.controller.ts (hunk 1 above)
Comment (stefanhar) [reply apps/services/temporary-voter-registry-api/src/app/modules/voterRegistry/voterRegistry.controller.ts:10]: I don't think the current timeframe allows for implementation of OAuth2, but I see where you are coming from.
Our goal is to present a working system by thursday.
After that deadline the user flows and functionality need to be tested probably accompanied by some iterations.
I'm confident we can implement auth into this service during that timeframe

**Review by eirikurn — APPROVED** [review]

File: apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+        },
+        national_id: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+        region_number: {
```
Comment (arnorhs) [inline apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js:19]: hvað er region number og name?

File: apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        region_name: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+        version: {
+          type: Sequelize.INTEGER,
+          allowNull: false,
+        },
+        created: {
+          type: Sequelize.DATE,
```
Comment (arnorhs) [inline apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js:32]: mundi maður ekki vilja datetime á þetta frekar? eða timestamp eða hvað sem það heitir

**Review by arnorhs — APPROVED** [review]

File: apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js (hunk 2 above)
Comment (stefanhar) [reply apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js:19]: Region number is a numeric representation of region name assigned by the national registry.
They represent what voting region (kjördæmi) the requested user belongs to

**Pull request merged**